test/test_Scene.py

```diff
@@ -83,5 +83,18 @@ def test_features(self):
             scene.get_feature_config('test3'),
             {'value': 'auto_value', 'another_value': None})
 
+    def test_indexing(self):
+        prim0 = draw.Spheres()
+        prim1 = draw.ConvexPolyhedra()
+
+        scene = draw.Scene([prim0, prim1])
+
+        self.assertIs(scene[0], prim0)
+        self.assertIs(scene[-1], prim1)
+        self.assertEqual(scene[:3], [prim0, prim1])
```

Comment:
I'd like to add one more test line:
```suggestion
        self.assertEqual(scene[:2], [prim0, prim1])
        self.assertEqual(scene[:3], [prim0, prim1])
```